test/integration.sh

```diff
@@ -9,7 +9,16 @@ set -euo pipefail
 result=1
 trap 'kill $(jobs -p); exit $result' EXIT
 
-(./tmp/bin/dex serve ./test/config/dex.yaml) &
+OBSERVATORIUM=${OBSERVATORIUM:=./observatorium}
+DEX=${DEX:-dex}
+PROMETHEUS=${PROMETHEUS:-prometheus}
+THANOS=${THANOS:-thanos}
+LOKI=${LOKI:-loki}
+OPA=${OPA:-opa}
+UP=${UP:-up}
+WEBSOCAT=${WEBSOCAT:=websocat}
```

Comment:
You can eliminate passing this variables here by sources `variables.env` here.

Reply:
That would be super awesome. However we have an issue concerning that https://github.com/bwplotka/bingo/issues/40. I haven't found enough motivation to fix it yet. cc @bwplotka